web-ui/src/main/resources/catalog/js/admin/ThesaurusController.js

```diff
@@ -253,7 +253,7 @@
       $scope.computeThesaurusNs = function() {
         $scope.thesaurusSuggestedNs =
             location.origin +
-            '/thesaurus/' + $scope.thesaurusSelected.dname + '/' +
+            '/thesaurus/' + $scope.thesaurusSelected.dname.replace(/[^\d\w]/gi, '') + '/' +
```

Comment:
I would maybe at least allow ".-_" which are quite often used in thesaurus filename?

Reply:
+1 can you suggest a file change?

Reply:
\w already include "_"  So I believe the suggestions was to change the regex to the following.

[^\d\w-\.]